doc/orchestrator.rst

```diff
@@ -543,12 +551,75 @@ When you run the experiment, the following output will appear.
   | 2  | single_shard_db_identifier_0 | DBNode        | 1538905.5   | 0       | 73.1798 | Cancelled | 0            |
   | 3  | multi-shard-db-identifier    | DBNode        | 1538905.6+2 | 0       | 49.8503 | Cancelled | 0            |
 
-Source Code
------------
+How to Run the Example
+----------------------
+Below are the steps to run the experiment. Find the
+:ref:`experiment source code<Experiment Source Code>`
+and :ref:`application source code<Application Source Code>`
+below in the respective subsections.
 
-Experiment script
-^^^^^^^^^^^^^^^^^
-.. code-block:: python
+Step 1 : Setup your directory tree
+    Your directory tree should look similar to below::
+
+      Multi-db-example/
+        SmartSim/
+        SmartRedis/
```

Comment:
I would suggest moving SmartSim and SmartRedis to the top level, not within an experiment. One installation of each should be reusable across experiments.

Reply:
I see!